pages/contribution-flow.js

```diff
@@ -202,7 +202,16 @@ class NewContributionFlowPage extends React.Component {
       return <ErrorPage error={error} />;
     }
 
-    return <Page {...this.getPageMetadata()}>{this.renderPageContent()}</Page>;
+    return (
+      <Page
+        {...this.getPageMetadata()}
+        showFooter={false}
+        menuItems={{ solutions: false, product: false, company: false, docs: false }}
```

Comment:
If I do this, will the product menu be displayed?

```jsx
menuItems={{ solutions: false, product: true, company: false, docs: false }}
```

Reply:
Yes, that is correct. In the current design discussion we had we are planning to hide the nav bar and footer on the contribution flows; except for that the nav-bar stays the same everywhere else. In the future we will hide the nav bar in the signup flow as well. So this kinda fine tuning to display or not display each menu item might be an overkill but I just added it in case this changes in the future.

Reply:
It's fine if we don't want to allow fine-tuning, but then this prop is misleading as it let developers think they could customize each entry individually. An alternative if you did not want to support this would have been something like:

```jsx
menuItemsView={'FULL' | 'SIMPLIFIED'}
```